skyportal/handlers/api/gcn.py

```diff
@@ -1954,3 +1957,58 @@ class Validator(Schema):
         except Exception as e:
             return self.error(f"Error generating summary: {e}")
         return self.success(data=contents)
+
+
+class LocalizationDownloadHandler(BaseHandler):
+    @auth_or_token
+    async def get(self, dateobs, localization_name):
+        """
+        ---
+        description: Download a GCN localization skymap
+        tags:
+          - localizations
+        parameters:
+          - in: path
+            name: dateobs
+            required: true
+            schema:
+              type: dateobs
+          - in: path
+            name: localization_name
+            required: true
+            schema:
+              type: localization_name
+        responses:
+          200:
+            content:
+              application/json:
+                schema: LocalizationHandlerGet
+          400:
+            content:
+              application/json:
+                schema: Error
+        """
+
+        with self.Session() as session:
```

Comment:
I'd suggest wrapping the handler or at least the content of the context manager in a try-except statement to avoid crashes

Reply:
yup.